django/core/signing.py

```diff
@@ -255,6 +258,11 @@ def unsign_object(self, signed_obj, serializer=JSONSerializer, **kwargs):
 
 
 class TimestampSigner(Signer):
+    def __init__(self, *args, **kwargs):
+        self.expiration_key = kwargs.pop("expiration_key", None)
+        self.payload_timestamp = None
```

Comment:
Changes to the `TimestampSigner` should be expressed in [docs](https://docs.djangoproject.com/en/stable/topics/signing/#verifying-timestamped-values).